api/KS.FiksProtokollValidator.WebAPI/TestCases/no.ks.fiks.arkiv.v1/HentJournalpostF1/testInformation.json

```diff
@@ -1,5 +1,5 @@
 {
-    "testName": "Hent Journalpost - Feilsituasjon 1 - Ugyldig SystemID som nøkkel",
+    "testName": "Hent Journalpost - Feilsituasjon 1 - Ikkefunnet - Ikke eksisterende SystemID som nøkkel",
```

Comment:
ikke funnet i to ord? eller er det en nøkkel?

Reply:
Det er en feilmeldingstype tilsvarende ugyldigforespørsel